examples/hosts/app-integration/external-data/src/view/debugView.tsx

```diff
@@ -57,35 +60,47 @@ export interface IDebugViewProps {
  * For the purposes of this test app, it is useful to be able to see both data sources side-by-side.
  */
 export const DebugView: React.FC<IDebugViewProps> = (props: IDebugViewProps) => {
+	const { model, unresolvedChanges } = props;
+	// Flag that represents the state in which clients are actively fetching external data.
+	const [fetchingExternalData, setFetchingExternalData] = useState(false);
 	return (
 		<div>
 			<h2 style={{ textDecoration: "underline" }}>External Data Server App</h2>
-			<TaskListView model={props.model} />
-			<ExternalDataView />
-			<SyncStatusView />
-			<ControlsView model={props.model} />
+			<ExternalAppForm model={model} setFetchingExternalData={setFetchingExternalData} />
+			<ExternalDataView setFetchingExternalData={setFetchingExternalData} />
+			<SyncStatusView
+				fetchingData={fetchingExternalData}
```

Comment:
It looks like here you're having the SyncStatusView reflect the sync status of the debug external data view, but I don't think we need to worry about showing that -- it's the sync status of the FluidData that we want to highlight.  LMK if I'm misreading tho

Reply:
Yes, to confirm. For both the incoming and outgoing unsync'd changes, the resolution UI is enough. The "resolving changes" debug message further reinforces this status.

 Other cases such as idle or fetching is reflected by the current state of the code. The SyncStatusView reflects the status of Fluid data, whether idle, currently fetching external data or resolving local conflicts.